docs/user-guide/ci.rst

```diff
@@ -0,0 +1,74 @@
+.. role:: bash(code)
+   :language: bash
+
+CI Configuration
+================
+
+Follow these steps to configure CI for GitHub Actions on your repository:
+
+* Step 1:
+   Add the Certora CLI key `as a secret`_ to your repository, the secret name is "CERTORAKEY"
+   and the value is the key provided.
+
+* Step 2:
+   Create a :bash:`certora_verification.yml` file under :bash:`.github` directory and 
+   use the following `.yaml` example as a template you can use to start running `certora-cli`.
+
+.. code-block:: yaml
+
+   name: certora-verification
+
+   on:
+   push:
+      branches:
+         - main
+   pull_request:
+      branches:
+         - main
+
+   workflow_dispatch:
+
+   jobs:
+      verify:
+         runs-on: ubuntu-latest
+
+         steps:
+            - uses: actions/checkout@v4
+            with:
+               submodules: recursive
+
+            - name: Install python
+            uses: actions/setup-python@v2
+            with: { python-version: 3.9 }
+
+            - name: Install java
+            uses: actions/setup-java@v1
+            with: { java-version: "11", java-package: jre }
+
+            - name: Install certora cli
+            run: pip3 install certora-cli==7.0.7
+
+            - name: Install solc
+            run: |
+               wget https://github.com/ethereum/solidity/releases/download/v0.8.23/solc-static-linux
+               chmod +x solc-static-linux
+               sudo mv solc-static-linux /usr/local/bin/solc8.23
```

Comment:
This isn't standard. Why not instruct to use `solc-select`?
We can put this commented out and above it put a comment "If your project depends on compiling with multiple `solc` versions, you can install and differentiate them using these commands"

Reply:
@shoham-certora @Roy-Certora do you know if using "solc-select" works better for most of the customers than using a specific version ?

Reply:
@shellygr i also think using solc-select is better, i added it instead of wget and and also added the comment above it